packages/ui/components/NotificationBanner/NotificationBanner.spec.tsx

```diff
@@ -61,3 +61,17 @@ describe('NotificationBanner Component', () => {
     })
   })
 })
+
+test('renders rich text when richText prop is true', () => {
```

Comment:
both these tests are doing the same thing, you are passing html in this, not rich text `<p>Test Content</p>`

Reply:
wrap with  `<RichTextRenderer>` to make sure this is rendering/working properly